modules/webgl/src/classes/vertex-array.js

```diff
@@ -114,7 +114,7 @@ export default class VertexArray {
     this.drawParams = this.drawParams || this._updateDrawParams();
 
     // Override with any application supplied draw parameters
-    return Object.assign({}, this.drawParams, appParameters);
+    return this.drawParams;
```

Comment:
Remove comment, and also remove argument `appParameters` given its not used any more.